tests/ui-toml/wildcard_imports/clippy.toml

```diff
@@ -1 +1,2 @@
 warn-on-all-wildcard-imports = true
+ignored-wildcard-imports = ["utils"]
```

Comment:
One last thing actually, would be good to add a comment here that mentions it's ignored because of `warn-on-all-wildcard-imports`